bus-mapping/src/circuit_input_builder/withdrawal.rs

```diff
@@ -0,0 +1,62 @@
+//! Withdrawal & WithdrawalContext utility module.
+
+use eth_types::Address;
+
+use crate::Error;
+
+#[derive(Debug, Default)]
+/// Context of a [`Withdrawal`].
+pub struct WithdrawalContext {
+    /// Unique identifier of a withdrawal. This value starts from 0 and then increases
+    /// monotonically.
+    id: u64,
+
+    /// Reversible Write Counter tracks the number of write operations in the
```

Comment:
I just deleted this field since we don't need it. Fixed in [cad4ea4](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1634/commits/cad4ea47fc78e8b5f44451124629ba6b5bdff4b4)